apps/server/src/modules/school/service/school.service.spec.ts

```diff
@@ -90,80 +90,93 @@ describe('SchoolService', () => {
 		afterAll(() => {
 			schoolServiceSpy.mockRestore();
 		});
-		it('should call the createOrUpdateService', async () => {
-			const ret = await schoolService.saveProvisioningSchoolOutputDto(provisioningDto);
-			expect(ret).toBe(returnDO);
-			expect(schoolServiceSpy).toHaveBeenCalled();
+
+		describe('when provisioningDto is given', () => {
```

Comment:
in my opinion, you dont strictly need a when clause if there are no different cases. here I immediatly ask: "what if not?"